spec/schema.json

```diff
@@ -16,53 +16,346 @@
     }
   ],
   "paths": {
-    "/request": {
+    "/status": {
+      "get": {
+        "summary": "Returns information describing the strike-off status of the company",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "200": {
+            "description": "a Status object has been returned successfully",
+            "schema": {
+              "$ref": "#/definitions/StrikeOffStatus"
+            }
+          },
+          "404": {
+            "description": "not found"
+          }
+        }
+      }
+    },
+    "/": {
       "post": {
-        "summary": "Temporary end-point to prove sprint zero connectivity from the web app",
+        "summary": "Create a new strike-off objection request",
         "produces": [
           "application/json"
         ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "201": {
+            "description": "An objection request has been received.",
+            "schema": {
+              "$ref": "#/definitions/ResourceCreated"
```

Comment:
Could be more specific here - 'ObjectionCreated' maybe.

Reply:
the same object structure is returned by post attachment as well so I named it ResourceCreated for re-use. I could name it ObjectionCreated but I'd also have to create an AttachmentCreated as well.

Reply:
Ok. Fair enough. Confluence page had different field names being returned (objectionId and fileId) but I'm happy with the generic approach.